### ISSUE TITLE
Custom styles: Enable threat feedback for scaling, transparency, coloring separately

### ISSUE THREAD

**Issue by Backupiseasy** [Backupiseasy/ThreatPlates#395]
* https://www.curseforge.com/wow/addons/tidy-plates-threat-plates/issues/351
  * But one thing that always bothered me is that while you can customize specific enemy nameplates in custom nameplates, this also "disables" the threat on those nameplates unless you choose the option to use threat system. The thing is, I want to use custom nameplates for specific enemies and I want them to have a custom appearance like scale and color, but if I choose to use the threat system it automatically overrides the custom nameplates properties like the scale. I still want to see the threat, but just the color. I don't want it to change everything or override it via the threat system. It's hard to explain.... Hope you know what I mean. Is there any way around this?
  * Also, when you use custom scale for specific enemies and they start to cast they use the scale from the scale menu, so tiny nameplates suddenly get huge depending on what scale you set for casting enemies.

*  I think I understand what you mean. I guess I could add an option (or options) to allow you to enable only thread coloring/scaling/... Currently it's only an all or nothing type of thing. I want to rework custom nameplates anyway, but that will take some time.
* Regarding your second question: This means that situational scaling overwrites the custom scale of the custom nameplate. Hm, I have to look at this and think about if that's correct behaviour or not. 
* The casting scale issue only happens when you enable the threat system for custom nameplates, correct?
   * Yeah, only when I enable threat system.
   * An example: I want the small maggots in WM to have small nameplates, so I use custom nameplates for them. But I also have casting scale enabled in general options to make the nameplates of casting units bigger. So what happens is that the maggots have really small nameplates but once they start to cast something they get huge, like other units standart size + casting size on top of it. So you have to turn off threat system for those nameplates but well, you can't see their threat anymore. I only use coloring in the threat system.
   * In short, I want to have small nameplates on specific units, and be able to see their threat (color) but I don't want them to change their scale or anything else really, no matter what. So, like you said, more options in custom nameplates would be awesome, even better if other settings don't override those options.
